docs/docs/python/goals/fmt.mdx

```diff
@@ -16,4 +16,4 @@ See [here](../overview/linters-and-formatters.mdx) for how to opt in to specific
 - Pyupgrade
 - yapf
 
-If you activate multiple formatters, Pants will run them sequentially so that they do not overwrite each other. You may need to update each formatter's config file to ensure that it is compatible with the other activated formatters.
+If you activate multiple tools, Pants will run them sequentially so that they do not overwrite each other. You may need to update each tools' config file to ensure that it is compatible with the other activated tools.
```

Comment:
```suggestion
If you activate multiple tools, Pants will run them sequentially so that they do not overwrite each other. You may need to update each tool's config file to ensure that it is compatible with the other activated tools.
```